Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/TorsionForceField.inl

```diff
@@ -118,6 +120,28 @@ void TorsionForceField<DataTypes>::addKToMatrix(linearalgebra::BaseMatrix* matri
 	}
 }
 
+template <typename DataTypes>
+void TorsionForceField<DataTypes>::buildStiffnessMatrix(core::behavior::StiffnessMatrix* matrix)
+{
+    auto dfdx = matrix->getForceDerivativeIn(this->mstate)
```

Comment:
is this method returning a ref? Because I can see you are changing its value at the end of the method.

Reply:
Not really. It's a temporary small object which itself has a reference to the global matrix. So even if you change the small object which is then destroyed, you still modify the global matrix.